analyze.py

```diff
@@ -70,7 +69,7 @@ def is_fubar(results):
             has_wrong_pubkey = True
             logging.debug(conn['pubkey'][0] + ' is a fubar pubkey size')
             fubar = True
-        if ec_kex and pubkey_bits < min_ec_size:
+        if ec_kex and pubkey_bits < 160:
```

Comment:
There are no clients that would support 160 bit curves but not the 256 bit curves. Either they support no curves, just P-256 and P-384 or P-256, P-384 and P-521 or all curves. So even for the legacy config there's no need to reduce the size of parameters, in fact, using 256 bit curves there should be mandatory to maximize interoperability.

@jvehent can you fix the json's?